File: GO-BASIC/01.first-steps-in-coding/09.yard-greening/yard-greening.go
package main

import "fmt"

func main() {
	var yard float32

	fmt.Scanln(&yard)

	var price float32 = yard * 7.61

	var finalPrice float32 = price  * 0.82
	var discount float32 =  price * 0.18

	fmt.Printf("The final price is: %f lv.\nThe discount is: %f lv.", finalPrice, discount)
}

File: C++Basic/First-Steps-In-Coding-Lab/FirstStepsInCoding/yardGreening/yardGreening.cpp
#include <iostream>

using namespace std;

int main() {
	
	double yard;
	cin >> yard;

	double cost = yard * 7.61;
	
	double discount = cost * 0.18;

	double finalCost = cost - discount;
	
	cout << "The final price is: " << finalCost << " lv." << endl;
	cout << "The discount is: " << discount << " lv.";
	
	return 0;


}